### ISSUE TITLE
Error finding a topic by its tag with Postgres

### ISSUE THREAD

**Issue by ajmonfue** [tinode/chat#872]
### Subject of the issue
Error finding a topic by its tag with Postgres database

#### Server-side
- [x] Your own setup:
  * platform (Linux - Docker)
  * version of Tinode server: [v0.22.8](https://github.com/tinode/chat/releases/tag/v0.22.8)
  * database backend: postgres
  * standalone

#### Client-side
- [x] TinodeWeb/tinodejs: javascript client
  * Browser make and version: Version 113.0.5672.126 (Official Build) (x86_64)

### Steps to reproduce
1. Log in
2. Create a new topic with any tag
3. Search the topic with the specified tag

### Expected behaviour
You should find the topic

### Actual behaviour
You get a 500 error on the client

### Server-side log
```
W2023/06/24 01:47:45 topic[fndre3rpbWBWV4] meta.Get.Sub failed: can't scan into dest[4]: unable to assign to *int
```

### Client-side log
Error response on the socket, after the `{get} find` message:
```
{
    "id": "87006",
    "topic": "fnd",
    "code": 500,
    "text": "internal error",
    "ts": "2023-06-24T02:02:33.255Z"
}
```

**Comment by ajmonfue** [comment]
I think the error is caused by the line https://github.com/tinode/chat/blob/master/server/db/postgres/adapter.go#L2396 . Instead of an int, it should be a bool

**Comment by or-else** [comment]
Thank you. Fixed in https://github.com/tinode/chat/commit/1769c75e885cf70ad47aa9f47fdcb3b493677cd6

**Comment by or-else** [comment]
Released in 0.22.9
